### DISCUSSION TITLE
LegacyImports

### DISCUSSION THREAD

**Discussion by itmwiw** [renovatebot/renovate#26556] [Request Help]
### How are you running Renovate?

None

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Hello,
First of all, thanks for the effort—great work, everyone!
My question pertains to LegacyImports. I'm uncertain about the difference between setting it to true or false. I've discovered that it is somehow related to the absolute path in vendors, but setting it to false and true both seem to produce the same result when checking the vendor folder. Can you enlighten me, please?
Thanks in advance.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
I think you might have the wrong repo
